apps/percona/chaos/openebs_cstor-pool_network_delay/run_litmus_test.yml

```diff
@@ -0,0 +1,84 @@
+---
```

Comment:
lets create the directory as `openebs_cstor_pool_network_delay` rather than `openebs_cstor-pool_network_delay`

Reply:
you can remove openebs, and just use `cstor_pool_network_delay`.. and accordingly, you need to change all the files